### ISSUE TITLE
Predict correctly after training. But load the model and predict again will get a "no detections"

### ISSUE THREAD

**Issue by MikeHawkIsGold** [ultralytics/ultralytics#1229]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I use this to train and predict after finishing the train
```
    model = YOLO("yolov8n.yaml")  

    model.train(data="G:\BiShe\\ultralytics\\data.yaml", epochs=100)
    model.val()
    fileToUse = "runs\\crisDeadLift.jpg"
    results = model.predict(source=fileToUse,save=True)
    model.export(format="torchscript")
```

I can get a correct result.

But after that, I use that to predict:
```
    model = YOLO("yolov8n.torchscript")

    fileToUse = "runs\\crisDeadLift.jpg"
    results = model.predict(source=fileToUse,save=True) 
```
I can only get "no detections". Also happened on other images.
Any solutions?

**Comment by MikeHawkIsGold** [comment]
I think maybe the problem is my model. I tried the pre-trained model "yolov8n.pt" and it can work correctly. But I still don't know which step is wrong....

**Comment by MikeHawkIsGold** [comment]
After updating to version 8.0.49, the weird problem has been fixed.

**Comment by pderrenger** [comment]
@s3xysteak glad to hear the issue has been resolved with the latest update to version 8.0.49! If you encounter any further issues or have additional questions, feel free to ask. Happy detecting!

**Comment by glenn-jocher** [comment]
@pderrenger That's great news! 🎉 If anything else comes up, don't hesitate to reach out. Keep enjoying YOLOv8!
